meilisearch/client.py

```diff
@@ -23,7 +23,8 @@ def __init__(self, url, apiKey=None):
         apiKey : str
             The optional API key for MeiliSearch
         """
-        self.config = Config(url, apiKey)
+        self.config = Config(url, apiKey, timeout=timeout)
```

Comment:
```suggestion
        self.config = Config(url, apiKey, timeout=None)
```

The default value is `None`, so it should not throw an error if you add `None` as the value of the parameter.

Reply:
I got to default timeout updated to None.